dataverse-persistence/src/main/resources/db/migration/V79_add_depentent_validation_to_authorIdetifierScheme.sql

```diff
@@ -0,0 +1,2 @@
+update datasetfieldtype set validation = '[{"name":"standard_input"}, {"name":"required_dependant","parameters":{"context":["DATASET"], "dependantField": "authorIdentifier"}}]'
+where id  = 11 and name = 'authorIdentifierScheme';
```

Comment:
So that we are 200% safe. It is better not to update, than update to much or update a wrong record ,if for some reason id values change.

Reply:
But we want to catch them all, don't we? But not insisting because either way, it remains error prone. To be 100% correct one would need to ensure that there exists a sibling named `authorIdentifier` and instead of verifying by id ensure that the parent is `author` and that `author` doesn't have any parent himself.

That being said, don't we want to have a `required_dependant` on `authorIdentifier` as well? So that if either one is filled the other one needs to be filled as well.